### ISSUE TITLE
[ipcamera] ONVIF Not connecting to D-LINK 5222L Camera

### ISSUE THREAD

**Issue by protectivedad** [openhab/openhab-addons#13983]
## Expected Behavior
The camera's documentation and ONVIF website says the D-Link DCS-5222L conforms to "Profile S". I expected it would be recognized by the ipcamera binding.

## Current Behavior
The traced log shows
```
07:08:41.150 [DEBUG] [mera.internal.handler.IpCameraHandler] - About to connect to the IP Camera using the ONVIF PORT at IP:192.168.189.215:80
07:08:41.151 [TRACE] [camera.internal.onvif.OnvifConnection] - Sending ONVIF request:GetSystemDateAndTime
07:08:41.157 [TRACE] [camera.internal.onvif.OnvifConnection] - Onvif reply is:<?xml version="1.0" encoding="iso-8859-1"?>
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
         "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en">
 <head>
  <title>400 - Bad Request</title>
 </head>
 <body>
  <h1>400 - Bad Request</h1>
 </body>
</html>
```
## Possible Solution
The [ONVIF feature list for DCS-5222L](https://www.onvif.org/member-portal/wp-content/uploads/sites/2/2017/01/6352096695502834792XMLFileDoc.xml) doesn't list GetSystemDateAndTime. I rebuilt ipcamera with GetCapabilities first and GetSystemDateAndTime second. That didn't help:
```
07:36:13.066 [DEBUG] [mera.internal.handler.IpCameraHandler] - About to connect to the IP Camera using the ONVIF PORT at IP:192.168.189.215:80
07:36:13.067 [TRACE] [camera.internal.onvif.OnvifConnection] - Sending ONVIF request:GetCapabilities
07:36:13.072 [TRACE] [camera.internal.onvif.OnvifConnection] - Onvif reply is:<?xml version="1.0" encoding="iso-8859-1"?>
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
         "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en">
 <head>
  <title>400 - Bad Request</title>
 </head>
 <body>
  <h1>400 - Bad Request</h1>
 </body>
</html>
```

## Steps to Reproduce (for Bugs)
1. Create ONVIF thing pointed to camera IP with admin account and password.

## Context
I'm trying to get the pan and tilt to work.

## Your Environment
* Version used: OpenHAB-3.3.0
* Environment name and version:
 ```
 /usr/bin/java --version
openjdk 11.0.17 2022-10-18
OpenJDK Runtime Environment (build 11.0.17+8-post-Ubuntu-1ubuntu222.04)
OpenJDK 64-Bit Server VM (build 11.0.17+8-post-Ubuntu-1ubuntu222.04, mixed mode, sharing)
 ```
* Operating System and version:
```
lsb_release -a
LSB Version:    core-11.1.0ubuntu4-noarch:printing-11.1.0ubuntu4-noarch:security-11.1.0ubuntu4-noarch
Distributor ID: Ubuntu
Description:    Ubuntu 22.04.1 LTS
Release:        22.04
Codename:       jammy
```

**Comment by protectivedad** [comment]
After looking into it more I tested the camera using ONVIF Device Manager and the camera is recognised and functions properly. The mandatory GetSystemTimeAndDate is also working.

**Comment by Skinah** [comment]
See here for more info on the cause in the forum.
https://community.openhab.org/t/ipcamera-binding-cant-connect-to-d-link-dcs-8627lh/145438/9

I believe this may be solved in openHAB 4.0 Milestone 3 and newer as it will not require a response to GetSystemTimeAndDate anymore. You can test this under 3.x openHAB by using the custom built jar for Reolink users in the thread on the forum, so you should be able to check under both V3 and V4 right now, or wait a few days/week until the Milestone is released if you are fine to move to V4.

**Comment by Skinah** [comment]
Closing this issue as a user with a dlink camera has Co firmed the fix in the forum thread above.
